traitcuration/traits/datasources/clinvar.py

```diff
@@ -16,6 +16,19 @@
 NUMBER_OF_RECORDS = 200
 
 
+def run_clinvar():
+    """
+    This function initiates the process of downloading, parsing and storing ClinVar data
+    """
+    try:
+        download_clinvar_data()
+        traits_dict = parse_trait_names_and_source_records()
+        store_data(traits_dict)
+    except Exception as e:
+        print(f"Error: {e}")
```

Comment:
Does this print the entire stack trace, or just the exception type & message? Ideally we want the stack trace to be able to understand what happened.

Reply:
Agreed, dded stack trace printing